### DISCUSSION TITLE
Saving pages within a page via links 🙏🔥

### DISCUSSION THREAD

**Discussion by Woukim** [gildas-lormeau/SingleFile#1349] [Q&A]
**Is your feature request related to a problem? Please describe.**
It is very inconvenient if you need to save all the subpages that are contained on the whole page, because there are many of them, and each one you need to copy the link to then download...

**Describe the solution you'd like**
An option that allows you to automatically save subpages of a particular tab.

Please please, that would be extremely useful! ❤

**Comment by gildas-lormeau** [comment]
You can already do that. Press Ctrl/Cmd-A, then select in the context menu "SingleFile > Save selected links".
